src/molecules/formfields/GoogleAutoCompleteField/index.js

```diff
@@ -51,6 +51,18 @@ class GoogleAutoCompleteField extends Component {
   updateAddress = () => {
     const place = this.state.autocomplete.getPlace();
     const addressData = this.formatAddressInfo(place.address_components);
+
+    if (!addressData.number) {
+      const { value } = this.props.input;
+      const matches = value && value.match(/^(\d+)/);
```

Comment:
what are your thoughts on renaming this to something like `streetNumberMatch` just so it's clear what's being regex'ed for

Reply:
Yeah I like that, I'll make that change